introduction_to_amazon_algorithms/forecasting_services_comparison/sagemaker_example.ipynb

```diff
@@ -0,0 +1,1543 @@
+{
```

Comment:
<p>Spelling: <code>recieve</code> -&gt; <code>receive</code> (2 occurences in this notebook)  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3532/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>